bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/cmd/AntiPatternCommandParser.java

```diff
@@ -141,10 +152,27 @@ public Options getOptions() {
             Option.builder(REWRITE_SQL_FLAG_NAME)
                     .argName(REWRITE_SQL_FLAG_NAME)
                     .required(false)
-                    .desc("flag specifying if the queries should be rwwritten using an LLM")
+                    .desc("flag specifying if the queries should be rewritten using an LLM")
                     .build();
     options.addOption(rewriteSQLFlag);
 
+    Option llmBestEffortFlag =
```

Comment:
Just typing out loud, do we really need 2 flags?
I was thinking on keeping LLM_RETRIES_FLAG_NAME, if it has a value > 1 then we validate LLM_RETRIES_FLAG_NAME times. If it has a value of 1 we do not validate and return the result as is.

Reply:
So there is a difference, with llmBestEffortFlag if you actually do a prediction and it's wrong, it keeps the generated SQL even though is wrong, and you can retry 3 times if you want. but when it's off, then if the generated SQL is wrong, then it discards it, so it doesn't take the generated SQL into account, so you have this trade off between generating something and generating something of quality

Reply:
Changed this flag to instead of default to strict validation, it defaults to a best effor basis, esentially changing the flag to a strict validation flag